snap/snapcraft.yaml

```diff
@@ -53,6 +53,26 @@ parts:
     organize:
         "usr/bin/fuse-overlayfs": "libexec/rockcraft/fuse-overlayfs"
 
+  rockcraft-scripts:
+    source: ./snap/local
+    plugin: dump
+    organize:
+      craft.git: libexec/rockcraft/craft.git
+
+  git:
+    plugin: nil
+    stage-packages: [git]
+    build-attributes:
+      - enable-patchelf
+    prime:
+      - "usr/lib/git-core"
+      - "usr/share/git-core"
+      - "usr/lib/x86_64-linux-gnu"
+      # perl is part of the core22 / core24
+      - "-usr/lib/x86_64-linux-gnu/perl"
```

Comment:
I suspect these removals aren't needed; if you specify any inclusions when only those paths are primed

Reply:
I'll try to include only libcurl and see if that will work, then I could remove the exclusions entirely:
```
- "usr/lib/x86_64-linux-gnu/libcurl-gnutls*"
```

Reply:
In the end I needed to include the following to get rid of exclusion:
```yaml
      - "usr/lib/x86_64-linux-gnu/libcurl-gnutls*"
      - "usr/lib/x86_64-linux-gnu/libpsl*"
      - "usr/lib/x86_64-linux-gnu/librtmp*"
      - "usr/lib/x86_64-linux-gnu/libnghttp2*"
```